Makefile.in

```diff
@@ -27,10 +27,11 @@ SBOX_APPS = src/faudit/faudit src/fbuilder/fbuilder src/ftee/ftee
 SBOX_APPS_NON_DUMPABLE = src/fcopy/fcopy src/fldd/fldd src/fnet/fnet src/fnetfilter/fnetfilter
 MYDIRS = src/lib $(MAN_SRC)
 MYLIBS = src/libpostexecseccomp/libpostexecseccomp.so src/libtrace/libtrace.so src/libtracelog/libtracelog.so
+CONTRIB = src/zsh_completion/_firejail src/bash_completion/firejail.bash_completion
 MANPAGES = firejail.1 firemon.1 firecfg.1 firejail-profile.5 firejail-login.5 firejail-users.5
 SBOX_APPS_NON_DUMPABLE += src/fsec-optimize/fsec-optimize src/fsec-print/fsec-print src/fseccomp/fseccomp
 SECCOMP_FILTERS = seccomp seccomp.debug seccomp.32 seccomp.block_secondary seccomp.mdwx seccomp.mdwx.32
-ALL_ITEMS = $(APPS) $(SBOX_APPS) $(SBOX_APPS_NON_DUMPABLE) $(MYLIBS)
+ALL_ITEMS = $(APPS) $(SBOX_APPS) $(SBOX_APPS_NON_DUMPABLE) $(MYLIBS) $(CONTRIB)
```

Comment:
This breaks the build for me.
Error:
```
strip: src/zsh_completion/_firejail: file format not recognized
strip: src/bash_completion/firejail.bash_completion: file format not recognized
```
Reason:
```
install-strip: all
	strip $(ALL_ITEMS)
	$(MAKE) realinstall
```

Probably it would be the best to have an own shell-completion target (like man). IDK if your make skill can that (my not) or if we need help from a makefile expert.

Reply:
The problem with the install target is that it's not clear where it should be installed. For binaries there's common spots: $PREFIX is fine, but the zsh autocompletion is not necessarily there too. So I'd rather skip _firejail to be installed as worse than not installing it is installing it in the wrong place.
But let me see how I can fix the strip and install problem.

Reply:
Please check now. I never tried install, but it works for me now.